pkg/antctl/transform/networkpolicy/transform.go

```diff
@@ -63,35 +101,14 @@ func Transform(reader io.Reader, single bool, opts map[string]string) (interface
 	)(reader, single)
 }
 
-const sortByEffectivePriority = "effectivePriority"
-
-// Compute a tierPriority value in between the application tier and the baseline tier,
-// which can be used to sort all policies by tier.
-var effectiveTierPriorityK8sNP = (networkpolicy.DefaultTierPriority + networkpolicy.BaselineTierPriority) / 2
-
-type NPSorter struct {
-	networkPolicies []cpv1beta.NetworkPolicy
-	sortBy          string
-}
-
 func (nps *NPSorter) Len() int { return len(nps.networkPolicies) }
 func (nps *NPSorter) Swap(i, j int) {
 	nps.networkPolicies[i], nps.networkPolicies[j] = nps.networkPolicies[j], nps.networkPolicies[i]
 }
 func (nps *NPSorter) Less(i, j int) bool {
 	switch nps.sortBy {
 	case sortByEffectivePriority:
-		var ti, tj int32
-		if nps.networkPolicies[i].TierPriority == nil {
-			ti = effectiveTierPriorityK8sNP
-		} else {
-			ti = *nps.networkPolicies[i].TierPriority
-		}
-		if nps.networkPolicies[j].TierPriority == nil {
-			tj = effectiveTierPriorityK8sNP
-		} else {
-			tj = *nps.networkPolicies[j].TierPriority
-		}
```

Comment:
I don't understand why this is removed. I guess it would break the correctness of the effective order.

Reply:
I removed this because TierPriority can never be nil, reason being that when the network policies are processed if there is no TierPriority assigned then they are assigned DefaultTierPriority.

Reply:
Not true for K8s NetworkPolicy. You can test with K8s NetworkPolicy.

Reply:
got it